README.md

```diff
@@ -1,86 +1,118 @@
 # AMD MIGraphX
 
-AMD's graph optimization engine.
+AMD MIGraphX is AMD's graph inference engine to accelerate model inference on AMD GPUs. AMD MIGraphX can be used by
+installing binaries directly or building from source code.
+
+Note that all the following instructions are based on that ROCm has been installed successfully. ROCm installation
+instructions are explained in the [ROCm installation
+guide](https://rocmdocs.amd.com/en/latest/Installation_Guide/Installation-Guide.html).
+
+## Installing from binaries
+With ROCm installed correctly, MIGraphX binaries can be installed with the following command:
+```
+sudo apt update && sudo apt install -y migraphx
+```
+then the head files and libs are located at `/opt/rocm/include` and `/opt/rocm/lib`, respectively, which can be
+included and linked by adding the corresponding folders to the Makefile.
+
+## Building from source
+
+There are two ways to build the MIGraphX sources. One is installing the dependencies, then using 
+cmake to build the source. The other is using the ROCm build tool [rbuild](https://github.com/RadeonOpenCompute/rbuild).
+In the following, we will first list the dependencies required to build MIGraphX source code, then describe each of the
+two approaches.
+
+### List of dependencies
+The following is a list of dependencies required to build MIGraphX source. This list is also available in the
+requirement files `dev-requirements.txt` and `requirements.txt`.
 
-## Prerequisites
 * [ROCm cmake modules](https://github.com/RadeonOpenCompute/rocm-cmake) **required**
 * [MIOpen](https://github.com/ROCmSoftwarePlatform/MIOpen) for running on the GPU
+* [rocBLAS](https://github.com/ROCmSoftwarePlatform/rocBLAS) for running on the GPU
```

Comment:
You are also missing msgpack and json library in the list.

Reply:
added.